medium-tests/src/test/java/mediumtest/analysis/AnalysisMediumTests.java

```diff
@@ -1181,6 +1181,86 @@ void it_should_analyze_xml_file_in_non_ascii_directory_and_non_decoded_uri(Sonar
     assertThat(raisedIssueDto).isNotEmpty();
   }
 
+  @SonarLintTest
+  void it_should_trigger_analysis_after_analysis_properties_change(SonarLintTestHarness harness, @TempDir Path baseDir) {
```

Comment:
There is another test class that is called `AnalysisTriggeringMediumTests`, more suited for tests around triggering, I think it would be a better fit.

Reply:
Changed.